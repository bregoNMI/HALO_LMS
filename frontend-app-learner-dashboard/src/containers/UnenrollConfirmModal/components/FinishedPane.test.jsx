import React from 'react';
import { shallow } from '@edx/react-unit-test-utils';

import { FinishedPane } from './FinishedPane';

describe('UnenrollConfirmModal FinishedPane', () => {
  test('snapshot: gave reason', () => {
    const props = {
      gaveReason: true,
      handleClose: jest.fn().mockName('props.handleClose'),
    };
    expect(shallow(<FinishedPane {...props} />).snapshot).toMatchSnapshot();
  });
  test('snapshot: did not give reason', () => {
    const props = {
      gaveReason: false,
      handleClose: jest.fn().mockName('props.handleClose'),
    };
    expect(shallow(<FinishedPane {...props} />).snapshot).toMatchSnapshot();
  });
});
