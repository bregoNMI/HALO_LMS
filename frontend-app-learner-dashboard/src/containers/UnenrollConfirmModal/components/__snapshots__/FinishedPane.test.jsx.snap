// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnenrollConfirmModal FinishedPane snapshot: did not give reason 1`] = `
<Fragment>
  <h4>
    You are unenrolled
  </h4>
  <p>
    This course will be removed from your dashboard.
  </p>
  <ActionRow>
    <Button
      onClick={[MockFunction props.handleClose]}
    >
      Return to dashboard
    </Button>
  </ActionRow>
</Fragment>
`;

exports[`UnenrollConfirmModal FinishedPane snapshot: gave reason 1`] = `
<Fragment>
  <h4>
    You are unenrolled
  </h4>
  <p>
    Thank you for sharing your reason for unenrolling.  
    This course will be removed from your dashboard.
  </p>
  <ActionRow>
    <Button
      onClick={[MockFunction props.handleClose]}
    >
      Return to dashboard
    </Button>
  </ActionRow>
</Fragment>
`;
